cdap-app-fabric/src/main/java/co/cask/cdap/app/preview/PreviewServerModule.java

```diff
@@ -80,7 +80,7 @@ protected void configure() {
     bind(RuntimeStore.class).to(DefaultStore.class);
     expose(RuntimeStore.class);
 
-    bind(NamespaceQueryAdmin.class).to(DefaultNamespaceQueryAdmin.class).in(Scopes.SINGLETON);
+    bind(NamespaceQueryAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);
```

Comment:
Don't you need the same bindings as in `AppFabricServiceRuntimeModule`? Otherwise the DNA will miss the resource deleter.